### ISSUE TITLE
Windows Display Color Calibration: 'Finish' button incorrectly announced as 'unavailable'

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#19400]
### Brief summary

**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26200.7271)
Browser: Edge (Version 144.0.3719.18)
Screen Reader: NVDA (2025.3.2)

**Pre-Requisites:**
Enable NVDA 

**User Impact:**
Screen reader users may believe the Finish button is disabled and be unable to complete the calibration process independently. This can cause confusion, workflow interruption, and reliance on sighted assistance, reducing accessibility and usability.

### Steps to reproduce

1. Tab till start menu and hit Enter key.
2. Search for setting and hit Enter key to open it.
3. Tab till Display and hit Enter key.
4. tab till color profile and hit Enter key.
5. Tab till Calibrate Display button and hit Enter key.
6. "Welcome to Display Color Calibration" screen will appear, tab till Next button and hit Enter key.
7. Tab till Next button and press Enter on it until "You've successfully created new calibration" screen appear.
8. Observe that Screen reader (NVDA) is incorrectly announcing as unavailable on 'Finish' button on You've successfully created new calibration" dialog in windows or not.

### Actual behavior

Screen reader (NVDA) is incorrectly announcing as unavailable on 'Finish' button on You've successfully created new calibration" dialog in windows.
NVDA announces: Unavailable Finish Button Completes the wizard ALT + F

**Refer Attachment:**

1. Screen reader (NVDA) is incorrectly announcing as unavailable on 'Finish' button on You've successfully created new calibration" dialog in windows.png
2. Screen reader (NVDA) is incorrectly announcing as unavailable on 'Finish' button on You've successfully created new calibration" dialog in windows.mp4

https://github.com/user-attachments/assets/ae5a4c42-476c-405b-93c2-89534145fe32
<img width="979" height="624" alt="Image" src="https://github.com/user-attachments/assets/327f5fd4-16a2-4828-8823-f6e0f51e82e7" />

### Expected behavior

Screen reader (NVDA) is incorrectly announcing as unavailable on 'Finish' button on You've successfully created new calibration" dialog in windows.
NVDA should announce: Finish Button Completes the wizard ALT + F

### NVDA type

installed copy

### NVDA version

2025.3.2

### Windows version

Windows 11 version 25H2 (OS Build 26200.7271)

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs
